package.json

```diff
@@ -12,6 +12,7 @@
     "checks": "npm run lint & npm run typeCheck & npx vitest run",
     "build": "vite build",
     "build-internal": "vite build --base=/nucmorph-colorizer/ --config config/vite.internal-build.config.js",
+    "build-documentation": "vite build --base=/viewer/",
```

Comment:
Also threw this in for building for the colorizer-data documentation

Reply:
It's a build target for the Getting Started Guide, which contains a built copy of the viewer, stored in the subdirectory `viewer/`.

Reply:
I see - maybe there should be a comment that colorizer-data DEPENDS on this being here?